textalyze.js

```diff
@@ -17,7 +18,33 @@ function itemCounts(array) {
   // Your code here.
   // Run 'npm test' to see what tests need to pass.
 
+  array.forEach((value) => {
+    const sum = counts.get(value) ? counts.get(value) + 1 : 1;
+    counts.set(value, sum);
+  });
+
   return counts;
 }
 
-module.exports = { itemCounts };
+/**
+ * Takes an arbitrary String as input
+ * and returns an Array with the counting of all the characters in the string
+ * @param {String} arbitrary string
+ * @returns {Array} a Map with the counting of each char
+ */
+function stringCharsToArray(string) {
+  const arrayString = string.split('');
+  return itemCounts(arrayString);
+}
+
+/**
+ * Takes an arbitrary String as input and returns a "sanitized" string
+ * that replaces all upper-case letters with their lower-case equivalent
+ * @param {String}
```

Comment:
Include descriptions of the parameters in the jsdoc.

Reply:
Sorry, forgot about that